src/components/application_manager/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -647,6 +648,7 @@ void RegisterAppInterfaceRequest::SendRegisterAppInterfaceResponseToMobile() {
   // Sends OnPermissionChange notification to mobile right after RAI response
   // and HMI level set-up
   GetPolicyHandler().OnAppRegisteredOnMobile(application->policy_app_id());
+  (*notify_upd_manager)();
```

Comment:
@LuxoftAKutsan 
Okay, seems like this construction is unavoidable, skip note above.